featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -6,6 +6,7 @@
 from woodwork.logical_types import Boolean, BooleanNullable
 
 from featuretools import primitives
+from featuretools.entityset.entityset import LTI_COLUMN_NAME
```

Comment:
Link to Variable
https://github.com/alteryx/featuretools/blob/97b6fdc1724c84353b880fa10736b08a1fa5de9d/featuretools/entityset/entityset.py#L28